app/src/debug/google-services.json

```diff
@@ -0,0 +1,56 @@
+{
+  "project_info": {
+    "project_id": "zulip-dev-1280",
+    "project_number": "659545034376",
+    "name": "Zulip-dev"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:659545034376:android:cca646dc1688a132",
+        "client_id": "android:com.zulip.android.dev",
+        "client_type": 1,
+        "android_client_info": {
+          "package_name": "com.zulip.android.dev",
+          "certificate_hash": []
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "659545034376-5jflfug7e8u87rimjkaab187tulgi69c.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.zulip.android.dev",
+            "certificate_hash": "ADD020F1B9111D8B4D422853E2381216EB7EA32E"
+          }
+        },
+        {
+          "client_id": "659545034376-78hg5hmrvqr50nn53t0nme97ndhqjmvs.apps.googleusercontent.com",
```

Comment:
I created a separate Google project called "Zulip Dev" that's coupled with the debug signing key (commit [869bfca354c](https://github.com/zulip/zulip-android/pull/52/commits/869bfca354cd95f7d32a3ba4b584be8d96a652a5)).  Together (these client IDs plus a constant debug signing key) make it so that a debug build can get OAuth tokens from the Google APIs, without additional set up.  It'll fail on sync with the prod servers tho, you'll have to use the same client ID for the backend to get the whole flow working.